android/src/main/AndroidManifest.xml

```diff
@@ -1,14 +1,22 @@
+<!--
+  ~  AndroidManifest.xml
```

Comment:
We moved away from having the filename and the initial "PSPDFKit" line inside the copyright header. Please consider updating style of this header to match the others.